implementation/domain/logic/jbb-system-impl/src/main/java/org/jbb/system/impl/cache/logic/CachePropertiesPropertyListener.java

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
When cache properties are changed,  connectionToDatabaseEvent must be sent. It event triggers refreshing of beans responsible of db layer (Datasource bean, EntityManagerFactory bean with hibernate config etc.). This class is just sending connectionToDatabaseEvent when notice some changes in cache properties.